src/sentry/static/sentry/app/components/commitRow.tsx

```diff
@@ -69,6 +111,25 @@ const AvatarWrapper = styled('div')`
   margin-right: ${space(2)};
 `;
 
+const EmailWarning = styled('div')`
+  font-size: ${p => p.theme.fontSizeSmall};
+  line-height: 1.4;
+
+  a {
+    color: ${p => p.theme.textColor};
+    border-bottom: 1px dotted ${p => p.theme.textColor};
+  }
+`;
+
+const EmailWarningIcon = styled(InlineSvg)`
+  position: relative;
+  margin-left: -11px;
```

Comment:
Should this icon be centered below user avatar?

Reply:
the overlap to the bottom right is intentional. does it look off?